<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TaskDialog</class>
 <widget class="QDialog" name="TaskDialog">
  <property name="windowTitle">
   <string>Add New Task</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="headerLabel">
     <property name="text">
      <string>Add New Task</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="styleSheet">
      <string>font-size: 18px; font-weight: bold;</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="titleEdit">
     <property name="placeholderText">
      <string>Task Title</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextEdit" name="descriptionEdit">
     <property name="placeholderText">
      <string>Task Description</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="priorityCombo">
     <item>
      <property name="text">
       <string>Low</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Medium</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>High</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="statusCombo">
     <item>
      <property name="text">
       <string>Backlog</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>In Progress</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Blocked</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Done</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QDateEdit" name="dueDateEdit">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="displayFormat">
      <string>yyyy-MM-dd</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="createdDateLabel">
     <property name="text">
      <string>Created: -</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="buttonLayout">
     <item>
      <widget class="QPushButton" name="saveButton">
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>